mappings/net/minecraft/client/gui/screen/ingame/InventoryScreen.mapping

```diff
@@ -1,12 +1,4 @@
-CLASS net/minecraft/class_806 net/minecraft/client/gui/screen/ingame/InventoryScreen
-	FIELD field_3510 mouseX F
-	FIELD field_3511 mouseY F
-	METHOD <init> (Lnet/minecraft/class_1963;)V
-		ARG 1 player
-	METHOD method_2946 renderEntity (IIIFFLnet/minecraft/class_1752;)V
-		ARG 0 x
-		ARG 1 y
-		ARG 2 size
-		ARG 3 mouseX
-		ARG 4 mouseY
-		ARG 5 entity
+CLASS net/minecraft/class_799 net/minecraft/client/gui/screen/ingame/InventoryScreen
```

Comment:
Are you sure that this is the correct name? I would name this `AbstractInventoryScreen` as in old official yarn.